docs/contributor/count-contributions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Ensure your contributions are counted
+---
+
+## Sign The Contributor License Agreement
+If you already contribute some PRs to [karmada-io](https://github.com/karmada-io), the CLA is signed.
+Otherwise, you should sign the agreement by following [Sign The CLA](https://github.com/kubernetes/community/blob/master/CLA.md).
```

Comment:
Seems this process is triggered by PR, but we don't require that, so where are contributors get start?

Reply:
cc @RainbowMango, should we push this forward？